tests/test_crypto.py

```diff
@@ -0,0 +1,101 @@
+
```

Comment:
I would prefer to treat the CLI commands like black boxes and ensure the various expected inputs produce the appropriate outputs rather than testing through running the various methods we expect the CLI commands to be calling in the order we expect them to be called in.  I have a few best practices around this I'm happy to share if you want input.